README.md

```diff
@@ -21,7 +30,7 @@ University of Groningen, Center for Information Technology, 2022
 
 ### Python
 
-- Python 3 (>= 3.6)
+- Python 3 (>= 3.10)
```

Comment:
Have we decided on version 3.10 over 3.9?  I think the lower we set the version, provided it's compatible, the better.  It gives the user and potential developer greater flexibility.

To be clear, I have no problem with using this version of Python.  I just wanted to point out there are advantages to providing the broadest choice.

Reply:
I changed the Doc to python 3.9